app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -251,12 +251,12 @@ def flavors
         end
 
         persister.flavors.build(
-          :type        => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SAPProfile",
-          :ems_ref     => value.profile_id,
-          :name        => value.profile_id,
-          :cpus        => value.cores,
-          :memory      => value.memory&.gigabytes,
-          :description => description
+          :type            => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SAPProfile",
+          :ems_ref         => value.profile_id,
+          :name            => value.profile_id,
+          :cpu_total_cores => value.cores,
```

Comment:
TODO: I should go through our parser and make sure all of these parameters have coverage in the refresher spec.